jedi/_compatibility.py

```diff
@@ -500,3 +501,16 @@ class Parameter(object):
         VAR_POSITIONAL = object()
         KEYWORD_ONLY = object()
         VAR_KEYWORD = object()
+
+
+class GeneralizedPopen(subprocess.Popen):
+    def __init__(self, *args, **kwargs):
```

Comment:
The signature of `Popen` is `args` and a bunch of named parameters so it would be better to set the parameters of the constructor to `args, **kwargs`.

Reply:
I disagree, I like the pattern of "add something"/"remove something" in these cases. :)